src/test/java/stepDefinitions/StepsSignUp.java

```diff
@@ -0,0 +1,125 @@
+package stepDefinitions;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.Assert;
+import pages.SignUpPage;
+import utils.FakerManager;
+
+import java.util.Collections;
+
+import static dictionary.ErrorMessages.*;
+
+public class StepsSignUp {
+
+    private SignUpPage singUpPage;
+
+    public StepsSignUp() {
+        this.singUpPage = new SignUpPage();
+    }
+
+    @And("the system shows me the registration form")
+    public void theSystemShowsMeTheRegistrationForm() {
+        Assert.assertEquals("The expected title was not obtained", "Sign Up", singUpPage.getTitleContainerPage());
```

Comment:
I would move these hardcoded strings to a constants file, what do you think?

Reply:
The titles and/or texts of the application to be validated are added to a class of constants. In the class of error messages the error texts that will be shown when the validations fail